### PR TITLE
Fix descriptions

### PR DESCRIPTION
**Pull request by PingDavidR**
- closes BRASS-1333

### CODE REVIEW

File: internal/resource/config/keypair/signing/import/key_pair_signing_import_data_source.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ type keyPairsSigningImportDataSourceModel struct {
 // GetSchema defines the schema for the datasource.
 func (r *keyPairsSigningImportDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	schemaDef := schema.Schema{
-		Description: "Describes a KeyPairsSigningImport.",
+		Description: "Describes the file for importing a key pair.",
```
Comment (wesleymccollam) [inline internal/resource/config/keypair/signing/import/key_pair_signing_import_data_source.go:58]: This should match our standard format, plus the description in the API doc - "Describes details of a signing key pair"

File: internal/resource/config/keypair/sslserver/import/key_pair_ssl_server_import_data_source.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ type keyPairsSslServerImportDataSourceModel struct {
 // GetSchema defines the schema for the datasource.
 func (r *keyPairsSslServerImportDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	schemaDef := schema.Schema{
-		Description: "Describes a KeyPairsSslServerImport.",
+		Description: "Describes a file for importing an SSL key pair.",
```
Comment (wesleymccollam) [inline internal/resource/config/keypair/sslserver/import/key_pair_ssl_server_import_data_source.go:58]: Same thing here but for "SSL server key pair"

File: internal/resource/config/license/license_data_source.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ type licenseDataSourceModel struct {
 // GetSchema defines the schema for the datasource.
 func (r *licenseDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	schemaDef := schema.Schema{
-		Description: "Describes a License.",
+		Description: "Describes a license summary object.",
```
Comment (wesleymccollam) [inline internal/resource/config/license/license_data_source.go:61]: I think you could remove the "object" here

**Review by wesleymccollam — CHANGES_REQUESTED** [review]

File: internal/resource/config/license/license_data_source.go (hunk 3 above)
Comment (PingDavidR) [reply internal/resource/config/license/license_data_source.go:61]: I actually went back and forth on including that word.

**Review by wesleymccollam — APPROVED** [review]

**Pull request merged**